README.md

```diff
@@ -242,6 +246,43 @@ string ask(call(get('find'),get('philosophers'),fun(let('name',get('$0')),call(g
 
 Here is the link to our [AskQL playground](http://cli.askql.org:3000/)!
 
+#### Developer info - how to run
+
+1. Copy .env.example to .env and set `PLAYGROUND_PORT`, `PLAYGROUND_ASK_SERVER_URL` and `PLAYGROUND_ASK_SERVER_VERSION_URL` appropriately.
```

Comment:
Changing this values shouldn't be required by default

Reply:
It's impossible to guess what URL for the backend the user would use.
The default will work with our Playground backend on cli.ask.org after I replace the old Playground with the new one.

Reply:
if you skip this step and run the server then the server gets variables from the env.example.